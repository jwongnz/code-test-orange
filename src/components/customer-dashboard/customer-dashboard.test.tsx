import React from 'react';
import CustomerDashboard from './customer-dashboard';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('CustomerDashboard', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <CustomerDashboard />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});