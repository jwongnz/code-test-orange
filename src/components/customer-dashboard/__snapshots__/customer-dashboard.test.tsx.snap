// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerDashboard renders correctly 1`] = `
<styled.div>
  <styled.div>
    <WithStyles(ForwardRef(Input))
      endAdornment={
        <ForwardRef(WithStyles)
          position="end"
        >
          <Memo />
        </ForwardRef(WithStyles)>
      }
      onChange={[Function]}
      placeholder="Search for customer"
      type="search"
      value=""
    />
    <WithStyles(ForwardRef(Button))
      color="primary"
      component={[Function]}
      to="/saveCustomer/"
      variant="contained"
    >
      Add Customer
    </WithStyles(ForwardRef(Button))>
  </styled.div>
  <Connect(CustomerList)
    searchTerm=""
  />
</styled.div>
`;
